ManageWikiNamespaces.php

```diff
@@ -58,6 +58,16 @@
 		'overridedefault' => false,
 		'requires' => [],
 	],
+	'wgNoFollowNsExceptions' => [
```

Comment:
Needs to be added to LocalSettings.php.

Reply:
Looking through LocalSettings.php, it doesn't seem like any of the ManageWikiNamespaces configs are there?

Reply:
Can you please still add this there? That's something that I'll look into later. It can't hurt to add it.

Reply:
Done.